docs/source/automated_feature_engineering/handling_time.rst

```diff
@@ -71,8 +71,18 @@ There is one row in the feature matrix corresponding to a row in ``cutoff_times`
                                       cutoff_time_in_index=True)
     feature_matrix
 
+It is often the case that we want our labels in our calculated feature matrix so that the ordering is consistent between the labels and the rows of the feature matrix. However, adding labels to the initial dataframe means that you would have to explicitly prohibit ``dfs`` from building features with that column. To bypass this, we can provide additional columns to cutoff times which will be added directly the feature matrix. While the two columns will be used as an index and cutoff time, any additional columns will appear as features in the resulting feature matrix. 
```

Comment:
do you think it is explicit enough that it is the _first_ two columns that get used and that order matters

Reply:
My understanding is that the first two columns get used but the order of those columns doesn't matter. How about the phrasing:
"While the first two columns will be used as index and cutoff time regardless of their order in the dataframe, any additional columns will appear as features in the resulting matrix."